src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js

```diff
@@ -88,6 +88,32 @@ Td.propTypes = {
   children: PropTypes.node,
 };
 
+const ParentIconStyle = styled.span`
+  display: flex;
```

Comment:
Not sure if these properties (display & direction) are necessary. As far as I can see, they don't provide a visual change.

Reply:
You are right! Thanks!